policy/policy_test.js

```diff
@@ -0,0 +1,9 @@
+import {PolicyGenerator} from './recipe2policy.js';
+import {SimpleRecipeProto} from './simplerecipe_test_proto.js';
+import {SimpleRecipe} from './simple_recipe.js';
+
+
+const policyGen = new PolicyGenerator(SimpleRecipe, "SimpleRecipe");
+const proto = policyGen.recipeToPolicy();
+console.assert(JSON.stringify(proto) == JSON.stringify(SimpleRecipeProto),
```

Comment:
What do you think about using mocha (or something similar if you prefer something else). Could do in a subsequent PR but I think we want it... pretty soon?

https://mochajs.org/

Reply:
Yeah, we should use somethings. Scott has a hand rolled testing framework, I might use that just because it requires no setup and basically means we have fewer deps.